lib/scrabble.rb

```diff
@@ -0,0 +1,64 @@
+# require_relative '../lib/scrabble_player'
+
+module Scrabble
+
+  class Scoring
+
+    attr_reader :SCOREBOARD
+
+    # constant that is a hash of the point values of the letters
+    SCOREBOARD = {
+      %w[a e i o u l n r s t]=>1,
+      %w[d g]=> 2, %w[b c m p]=>3,
+      %w[f h v w y]=> 4,
+      %w[k]=> 5,
+      %w[j x]=> 8,
+      %w[q z]=> 10
+    }
+
+    def self.score word
+
+      letter_values = []
+
+      raise ArgumentError.new "Error. Please enter a string." if word.class != String
+
+      raise ArgumentError.new "Error. Please enter a string of letters." if word.match(/^[[:alpha:]]+$/) == nil
+
+      raise ArgumentError.new "Error. Please enter a string that is less than 7 characters." if word.length > 7
```

Comment:
Should be `Error. Please enter a string that is less than or equal to 7 characters.`